engine/movie/intro.asm

```diff
@@ -325,8 +323,9 @@ PlayShootingStar:
 	call DelayFrames
 	farcall AnimateShootingStar
 	push af
+	; the call to LoadPresentsGraphic here was removed in localization
 	pop af
-	jr c, .next ; skip the delay if the user interrupted the animation
+	jr c, .next
```

Comment:
This is one of the best examples of a comment that shouldn't be deleted.

When we have gone on sprees of deleting comments, it's because we went overboard in the past with truly redundant comments, like:

```
    call LoadThing ; load the thing
```

But this comment about user interruption is _actual insight_ about the meaning of the carry flag at this point that would not be obvious from staring at this code if it weren't for the comment.